admin_scripts/aula_register.py

```diff
@@ -0,0 +1,205 @@
+elif chrome_version == '78':
+    driver_version = '78.0.3904.70'
+else:
+    print('Chrome version not supported.')
+    sys.exit(1)
+
+system_path = '/usr/local/bin'
+zipfile_name = driver_version + 'chromedriver_linux64.zip'
+zip_path = os.path.join(system_path, zipfile_name)
+extracted_filename = 'chromedriver'
+extracted_filepath = os.path.join(system_path, extracted_filename)
+
+# download gecko and setup
+if not os.path.isfile(zip_path):
+    try:
+        for fl in glob.glob(system_path + '/*chromedriver_linux64.zip'):
+            os.remove(fl)
+        for fl1 in glob.glob(system_path + '/chromedriver'):
+            os.remove(fl1)
+    except OSError:
+        pass
+
+    chromedriver_url = 'https://chromedriver.storage.googleapis.com/' + driver_version + '/chromedriver_linux64.zip'
+    wget.download(chromedriver_url, zip_path)
+
+    with zipfile.ZipFile(zip_path, 'r') as z:
+        z.extractall(system_path)
+
+    os.chmod(extracted_filepath, stat.S_IRWXU | stat.S_IXGRP | stat.S_IRGRP | stat.S_IXOTH)
+
+    print('Chromedriver downloaded and extracted to path: {}'.format(
+        extracted_filepath)
+    )
+else:
+    print('Chromedriver {} is already setup.'.format(driver_version))
+
+# start chrome headless
+opts = Options()
+# opts.set_headless()
+opts.add_argument('--headless')
+opts.add_argument('--no-sandbox')
+opts.add_argument('--disable-dev-shm-usage')
+# assert opts.headless  # Operating in headless mode
+browser = Chrome(chrome_options=opts, executable_path=extracted_filepath)
+wait = WebDriverWait(browser, timeout=10)
+try:
+    browser.get(url)
+except InvalidArgumentException as iae:
+    print('Invalid argument given: {} of type {}'.format(url, type(url)))
+    print(iae.message)
+    sys.exit(1)
+
+print('Chrome browser opened headless with url: {}'.format(url))
+register_code_list = list(register_code)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__11'))).send_keys('' + register_code_list[0] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__12'))).send_keys('' + register_code_list[1] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__13'))).send_keys('' + register_code_list[2] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__14'))).send_keys('' + register_code_list[3] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__15'))).send_keys('' + register_code_list[4] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__16'))).send_keys('' + register_code_list[5] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__17'))).send_keys('' + register_code_list[6] + Keys.ENTER)
+wait.until(expected.visibility_of_element_located((By.id, '__BVID__18'))).send_keys('' + register_code_list[7] + Keys.ENTER)
+
+try:
+    wait.until(expected.visibility_of_element_located((By.CLASS_NAME, 'aula-presence-overview')))
+except:
```

Comment:
Another minor tweak: use `except Exception:` so that we don't catch system errors as well. (This observation has been, er, borrowed from Dan: https://github.com/magenta-aps/os2datascanner/pull/29#discussion_r282418487.)